### ISSUE TITLE
Implement repeating gradients (repeating-linear-gradient, repeating-radial-gradient)

### ISSUE THREAD

**Issue by blacksvk** [servo/servo#10263]
Tested on Windows 8.1 x64. Test html: http://testujem.eu/servo/repeating-gradient.html

``` html
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Gradient test</title>
    <style>
        html{
            padding: 60px;
        }

        .gradient{
            height: 40px;;
            background: repeating-linear-gradient(45deg,darkblue 0,darkblue 22px,white 22px,white 44px,blue 44px,blue 66px,white 66px,white 88px);
        }
    </style>
</head>

<body>
    <div class="gradient"></div>
</body>

</html>
```

**Comment by mbrubeck** [comment]
https://drafts.csswg.org/css-images-3/#repeating-gradients

**Comment by pyfisch** [comment]
@highfive: assign me

This is a simple fix since webrender supports repeating gradients since a few days. The example works but I will wait until #16385 lands with a PR so I can add it for radial gradients too.

**Comment by pyfisch** [comment]
@emilio solved by #16385. Please close. Thanks!

**Comment by jdm** [comment]
Thanks for fixing it!
